schematools/introspect/geojson.py

```diff
@@ -0,0 +1,117 @@
+"""Converting a GeoJSON input file to Amsterdam Schema"""
+import json
+import re
+from decimal import Decimal as D
+from os.path import basename, splitext
+from typing import List
+
+from schematools.utils import ParserError
+from schematools.introspect.utils import DATASET_TMPL, TABLE_TMPL
+
+ID_FORMAT = re.compile("^([a-z0-9_]+)[/.](\d+)$", re.I)
+
+
+def introspect_geojson_files(dataset_id, files) -> dict:
```

Comment:
Hier zou wellicht nog wat opgeknipt kunnen worden om te voorkomen dat linting tools over complexity gaan vallen (maar dat geldt nog voor meer functies in schema-tools.

Reply:
Goede, ik heb de PR hierop aangepast